cmd/monitoring/main.go

```diff
@@ -5,12 +5,18 @@ import (
 
 	"github.com/gagliardetto/solana-go/rpc"
 	relayMonitoring "github.com/smartcontractkit/chainlink-relay/pkg/monitoring"
-	"github.com/smartcontractkit/chainlink-solana/pkg/monitoring"
 	"github.com/smartcontractkit/chainlink/core/logger"
+
+	"github.com/smartcontractkit/chainlink-solana/pkg/monitoring"
 )
 
 func main() {
-	coreLog := logger.NewLogger()
+	coreLog, closeLggr := logger.NewLogger()
```

Comment:
@vyzaldysanchez @samsondav Did we consider usages like this when implementing the debug disk logs?

@topliceanu Where does this command run?

Reply:
@vyzaldysanchez

Reply:
I don't think we considered this kind on usages when we implemented the disk logs. But I consider this will work just fine anyways, even for a one time usage, unless there are more things we would need to consider when the logger gets used like this.